bin/build-artifacts-in-docker.sh

```diff
@@ -1,37 +1,98 @@
 #!/usr/bin/env bash
 
-set -e
-set -o pipefail
-set -x
+set -eox pipefail
 
-ROOT_DIR=$(readlink -f $(dirname $0)/..)
+function usage() {
+  cat << EOF
+build-artifacts-in-docker.sh [--root_dir <dir>] [--base_image_name <name>] [--base_image_tag <tag>] [--online-dist] [--offline-dist]
+
+This script is used to build the offline and/or online distributions of presto-admin in the specified Docker image name.
+  [--root_dir <dir>]:         Directory on local filesystem where the presto-admin tree is located. Defaults to this
+                              script's current directory.
+  [--base_image_name <name>]: Name of Docker image to use during the build. Defaults to prestodev/centos6-presto-admin-tests.
+  [--base_image_tag <tag>]:   Tag of the Docker image to use during the build. Defaults to the tag stored in the base_images_tag.json file.
+  [--online-dist]:            Boolean argument indicating whether the online distribution should be build. If neither the
+                              --online-dist nor the --offline-dist flags are specified then both distributions are built.
+  [--offline-dist]:           Boolean argument indicating whether the offline distribution should be build. If neither the
+                              --online-dist nor the --offline-dist flags are specified then both distributions are built.
+EOF
+}
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+  case $key in
+      --root_dir)
+      ROOT_DIR="$2"
+      shift
+      shift
+      ;;
+      --base_image_name)
+      BASE_IMAGE_NAME="$2"
+      shift
+      shift
+      ;;
+      --base_image_tag)
+      BASE_IMAGE_TAG="$2"
+      shift
+      shift
+      ;;
+      --online-dist)
+      DIST_TARGETS="$DIST_TARGETS dist-online "
+      shift
+      ;;
+      --offline-dist)
+      DIST_TARGETS="$DIST_TARGETS dist-offline "
+      shift
+      ;;
+      *)    # unknown; show usage and error out
+      echo "Unknown option $key"
+      usage
+      exit 1
+      ;;
+  esac
+done
+
+if [[ -z "${ROOT_DIR}" ]]; then
+  ROOT_DIR=$(readlink -f $(dirname $0)/..)
+fi
 
 if [[ -z "${BASE_IMAGE_NAME}" ]]; then
   BASE_IMAGE_NAME="prestodev/centos6-presto-admin-tests"
 fi
 
-BASE_IMAGE_NAME=${BASE_IMAGE_NAME}-build
-
 if [[ -z "${BASE_IMAGE_TAG}" ]]; then
   BASE_IMAGE_TAG=$(cat ${ROOT_DIR}/base-images-tag.json | python -c 'import sys, json; print json.load(sys.stdin)["base_images_tag"]')
 fi
 
+if [[ -z "${DIST_TARGETS}" ]]; then
+  DIST_TARGETS="dist-online dist-offline"
```

Comment:
we have a `dist-online` target same as `dist`?

Reply:
Ya, `dist` is just a link to `dist-online`.